parser_test.go

```diff
@@ -642,3 +642,49 @@ func benchmarkSigning(b *testing.B, method jwt.SigningMethod, key interface{}) {
 		}
 	})
 }
+
+func TestNewNumericDateMarshalJSON(t *testing.T) {
```

Comment:
We should probably do that consistently at some point, but the convention seems to be `TestNumericDate_MarshalJSON`

Reply:
Also, this unit test should be in `types_test.go`

Reply:
renamed and moved in b72bb8e626a9dee1c43196840f2721af0b9ee424